go/test/endtoend/vreplication/cluster.go

```diff
@@ -81,9 +84,20 @@ type Tablet struct {
 	DbServer *cluster.MysqlctlProcess
 }
 
+func init() {
+	originalVtdataroot = os.Getenv("VTDATAROOT")
+}
+
 func initGlobals() {
-	vtdataroot = os.Getenv("VTDATAROOT")
+	rand.Seed(time.Now().UTC().UnixNano())
+	dirSuffix := 100000 + rand.Intn(999999-100000) // 6 digits
+	vtdataroot = path.Join(originalVtdataroot, fmt.Sprintf("vreple2e_%d", dirSuffix))
```

Comment:
I see you are taking no chances with collisions :smile:

Reply:
paranoia with a small basis in reality ;-)